drivers/ieee802154/ieee802154_cc2520.c

```diff
@@ -1312,7 +1312,8 @@ static int _cc2520_crypto_ccm(struct cipher_ctx *ctx,
 		return -EIO;
 	}
 
-	apkt->tag = apkt->pkt->out_buf + apkt->pkt->in_len;
+	memcpy(apkt->tag, apkt->pkt->out_buf + apkt->pkt->in_len,
```

Comment:
just remove that line entirely.

Reply:
cc2520 is not meant to be used as a generic ccm/uccm, but only for 15.4 security layer.

Anyway, I guess behaving properly as the crypto API defines, then le's keep this memcpy, however you'll need to test aptkt->tag before hand:
```
if (apkt->tag) {
        memcpy(...);
}
```

15.4 security layer sets it alway to NULL, so without this check you would get a fatal fault.

Reply:
Thanks